src/lib/utils/index.ts

```diff
@@ -379,3 +390,7 @@ export function deserializeError(se: SerializedError): Error {
 
   return e
 }
+
+export function isObject(o: any) {
```

Comment:
Lodash?

Reply:
Yep, thanks! I find their implementation very heavy, but let's not care for now (https://github.com/lodash/lodash/blob/master/isPlainObject.js).
Turns out lodash's method name is called `isPlainObject`, which is a great name for our type as well `PlainObject`!